sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -19,6 +19,16 @@ Install the Azure Text Analytics client library for .NET with [NuGet][nuget]:
 ```PowerShell
 dotnet add package Azure.AI.TextAnalytics
 ```
+> Note: This version of the client library defaults to the `v3.1-preview.5` version of the service.
+
+This table shows the relationship between SDK versions and supported API versions of the service:
+
+|SDK version|Supported API version of service
```

Comment:
This table does not render properly. Did we miss anything in the formatting?
![image](https://user-images.githubusercontent.com/7576220/120021188-97a7e480-bf9f-11eb-8b92-7c28b311c2d1.png)